include/cantera/numerics/Func1.h

```diff
@@ -265,6 +274,68 @@ class Pow1 : public Func1
     virtual Func1& derivative() const;
 };
 
+
+//! implements a tabulated function
+class Tabulated1 : public Func1
+{
+public:
+    Tabulated1(const std::vector<double>& tvec, const std::vector<double>& fvec,
```

Comment:
I am really conflicted about using raw pointers here. 

:thumbsup: for the docstring.

Reply:
It's not just the `Func1` family of classes, though. This applies to most Cantera classes, including `ThermoPhase`, `Kinetics`, `Transport`, and others. Switching broadly to `vector<double>` would incur a lot of extra temporary array allocations and copy operations, especially when using Cantera from languages other than C++. The performance impact for this class is almost certainly negligible, but in some other cases it will be more significant and I think the consistency is worthwhile.